CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+3.2.0
```

Comment:
3.4.0 i guess :)

Reply:
Why not `1.0`? This is a new bundle and I guess its release cycle will be independent from the Symfony release cycle.

Reply:
The ACL component also was released with the 3.x releases for compatibility reasons. I can imagine that it makes sense to release a 3.4 and 4.0 and then just let them go

Reply:
Well, we had the `symfony/security-acl` package before. But there is nothing like an AclBundle yet.

Reply:
Makes sense, but Then I'd rather have a `0.9` of sorts and have `1.0` be released without the deprecations that would be fixed in 4.0

Reply:
People are upgrading _from_ SF _to_ new acl bundle. Id say this requires the deprecationtrigger in SF, which also checks this bundle is enabled. As people are not using acl-bundle today, only security-bundle.

Reply:
The problem with this is, that if they install the ACL bundle and copy the configuration, things will break, as it's efficiently going from 3.3 to 4.0. The path you explained only works if developers fix the deprecations _before_ installing the bundle. However, the deprecation for the bundle configuration will be one of the first they see, due to it being in the container build process, rather than run-time.

Reply:
Agree portability increases DX :+1:

Reply:
Well, together with the old, a new version would be released I suppose, that means that 3.4 and 4.0 would be released at the same time for this bundle, including deprecation fixes.

The version number doesn't concern me, the DX and compatibility with 3.4 does

Reply:
What are the deprecations in this bundle? I don't see any.